### ISSUE TITLE
Automatically copy picked colour and keep Pika hidden

### ISSUE THREAD

**Issue by gitnixpie** [superhighfives/pika#64]
Most of the time I need to pick a colour quickly and paste the code where I need it without extra steps involved. Can we have an option for that?

**Comment by superhighfives** [comment]
Thanks for the suggestion! Will leave this one open to see how much interest there is for a feature like this.

**Comment by superhighfives** [comment]
My concern for this one was that there's a possibility that the user could have their pasteboard unexpectedly overwritten, so I'm leaning toward it being opt in. Could also consider a “tasteful” toast that notifies the user that `#f1dbce White Truffle` was copied to your clipboard so that the behaviour is expected / acknowledged.

From a conversation on tasteful toasts:
> I can be easily annoyed by the default MacOS notifications. Like, when 1Password tells me that a “One-time password has been copied” or “deleted” from my clipboard. It’s useful but it causes too much alarm in my brains.

**Comment by jackmcconnell** [comment]
Coming from ColorSnapper, having this as an option (default off) would be ideal.

**Comment by superhighfives** [comment]
Yeah, this seems like a good candidate for the next thing I work on. 👍

**Comment by superhighfives** [comment]
Thanks for the suggestions on this one, everyone. Really appreciate it. This is now going out as a part of https://github.com/superhighfives/pika/releases/tag/0.0.12.
